src/python/README.md

```diff
@@ -0,0 +1,51 @@
+# Python Scripts for Merge Conflict Analysis
+
+
+
+
+This directory contains Python scripts designed to facilitate the analysis of merge conflicts using various merge tools. The scripts allow users to recreate merges, analyze conflicts, and compare different merge algorithms' effectiveness.
+
+
+
+
+## Scripts Overview
+
+
+
+
+- `diff3_analysis.py`: This script analyzes merge conflicts for two merge tools on a given conflict. The tool that failed to merge should come first as an argument.
+
+
+
+
+## Prerequisites
+
+
+
+
+- Python 3.x installed on your system.
```

Comment:
Yes, the setup is the same as the rest of the project, we still are using a conda or mamba environment. I think I will just delete this line since there is already another README that more thoroughly explains setup. Thanks!